README.md

```diff
@@ -1 +1,37 @@
-# bubble-sort
\ No newline at end of file
+![alt text](https://camo.githubusercontent.com/3a5835d4f56c57cec85939ac345e43fef164c178/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f4d6963726f76657273652d626c756576696f6c6574 "Microverse")
+
+# Bubble Sort Exercise
+In this project we implemented a simple Bubble Sort exercise in Ruby, using Yield.
+We did the project in pair programming.
+
+## Built With
+Ruby
+
+## Live Demo
+[Live Demo Link](https://fanger53.github.io/bubble-sort/ "Live Demo")
```

Comment:
- [ ] Please fix your broken link. This link doesn't work because this is not a web page. For showing a live demo link, you can use repl.it or gitpod.io to display your projects. **Note:** For gitpod, you just have to append `gitpod.io/` to the URL of your project: https://gitpod.io/github.com/Fanger53/bubble-sort/tree/features.

- [ ] Please make sure to add setup instructions so that anyone can use your repo (`git clone` ... `ruby bubble_sort.rb`)